frontend/geeta-chopra-award-explorer/index.html

```
@@ -0,0 +1,354 @@
+            </div>
+        </section>
+    </main>
+
+    <footer class="gc-footer">
+        <p>Part of the <a href="../../index.html">Incredible India Explorer</a>. Built as an educational page on India's awards and honours.</p>
+    </footer>
+
+    <!-- Lightbox Modal -->
+    <div class="lightbox-modal" id="lightbox-modal" role="dialog" aria-modal="true" aria-label="Image viewer" hidden>
+        <button class="lightbox-close" id="lightbox-close" aria-label="Close image viewer">×</button>
+        <img id="lightbox-img" src="" alt="" />
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)